test/jdk/sanity/client/SwingSet/src/ButtonDemoTest.java

```diff
@@ -33,7 +35,7 @@
 import static org.testng.AssertJUnit.*;
 import org.testng.annotations.Test;
 import org.jemmy2ext.JemmyExt.ByToolTipChooser;
-import static org.jemmy2ext.JemmyExt.EXACT_STRING_COMPARATOR;
+import static org.jemmy2ext.JemmyExt.*;
```

Comment:
Looks like the change in this file is unnecessary? Note, we do not use '*' in the imports(same note in the EditorPaneDemoTest.java )

Reply:
reverted the change of that file